pkgs/tools/package-management/rpm/default.nix

```diff
@@ -29,6 +29,10 @@
   bubblewrap,
   autoconf,
   gnupg,
+
+  # Disable the unshare RPM plugin, which can be useful if
+  # RPM is ran within the Nix sandbox.
+  disableUnshare ? false,
```

Comment:
I think I would argue this should be disabled by default.

Reply:
I am hesitant to change the default given that this'd break the current default behaviour that people might be depending upon. (Even though I doubt that, as I see package managers from Nix to be used rarely)